src/schema/v2/user.ts

```diff
@@ -10,6 +10,7 @@ import { InternalIDFields } from "./object_identification"
 import { LocationType } from "schema/v2/location"
 import { ResolverContext } from "types/graphql"
 import { connectionWithCursorInfo } from "./fields/pagination"
+import CollectorProfile from "./me/collector_profile"
```

Comment:
feels a bit bad to import this from `me`, but I'm not seeing a good pattern in MP for "two different types use this same type/resolver combo".

We could:
- Copy/paste from `me` to `user` so that they can change independently (meh)
- Create a new folder `shared` for things shared by multiple base types
- Leave as-is

Any feelings @mc-jones?

Reply:
Agree that it feels a bit a anti-patern. I believe the way we generally handle this is to resolve the loader itself, similar to how do in [IdentityVerifications](https://github.com/artsy/metaphysics/blob/dd15c1c8176a3464dbf6b852e47e77625b669880/src/schema/v2/identityVerification.ts#L172-L178).

Reply:
I ended up creating a separate `collectorProfile` folder/type and referencing it in both places - I'm happy with how it turned out, but lmk if you think it's the wrong approach @mc-jones !

Reply:
I think this approach makes sense given the `collector_profile/:id` endpoint!